core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1296,14 +1296,14 @@ public void run() {
 
                     if (playerStatus == PlayerStatus.PLAYING) {
                         PendingIntent pauseButtonPendingIntent = getPendingIntentForMediaAction(
-                                KeyEvent.KEYCODE_MEDIA_PAUSE, numActions);
+                                KeyEvent.KEYCODE_MEDIA_PLAY_PAUSE, numActions);
```

Comment:
Why do you have to change the intents? Shouldn't it be enough to set `reinit` to false in the following section?

```
case KeyEvent.KEYCODE_MEDIA_PAUSE:
if (status == PlayerStatus.PLAYING) {
    mediaPlayer.pause(!UserPreferences.isPersistNotify(), true);
}
return true;
```